ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -77,7 +77,8 @@ yum install -y git
 mkdir -p /tmp/artifacts
 
 if [ ! -e dev-scripts ] ; then
-  git clone https://github.com/openshift-metal3/dev-scripts.git
+  git clone --branch must-gather --single-branch https://github.com/stbenjam/dev-scripts.git
```

Comment:
Temporarily pointing at https://github.com/openshift-metal3/dev-scripts/pull/983 to test it

Reply:
Why are you downloading this content on demand? You have full control over the content in this image. You may even run the step in a derivative of the `src` pipeline image where appropriate source is cloned. Please add the content you need to your image (statically), instead.

Reply:
This image === the `from:` stanza in this step.

Reply:
So git clone dev-scripts into the baremetal-installer image?

This gets run on the packet server, we'd end up cloning and then copying the data, instead of cloning it directly on a packet server. It'd increase our already long build time :-\

Reply:
Oh, I see -- this is a clone happening the server itself? Why? Why can't you run the script over `SSH` -- `ssh foo@bar bash -s < ./local.sh`

Reply:
It's about 6 different scripts, and 2 ansible playbooks.  We create an emulated baremetal infrastructure on 1 server using VM's and software called VBMC, as we as multiple internal networks.